README.md

```diff
@@ -36,7 +36,19 @@ Python package to develop applications with the Dispatch platform.
 
 ## What is Dispatch?
 
-Dispatch is a platform for developing scalable & reliable distributed systems.
+Dispatch is a cloud service for developing scalable and reliable applications in
+Python, including:
+
+- **Event-Driven Architectures**
+- **Background Jobs**
+- **Transactional Workflows**
+- **Multi-Tenant Data Pipelines**
+
+Dispatch differs from alternative solutions by allowing developers top write
```

Comment:
```suggestion
Dispatch differs from alternative solutions by allowing developers to write
```

Reply:
Hmm.. This conversation is marked as resolved, but I still see "top" instead of "to". Was the suggestion applied?

Reply:
Thanks for double-checking, probably messed up the rebase 👍 fixing.